front50-core/src/test/groovy/com/netflix/spinnaker/front50/model/application/ApplicationModelSpec.groovy

```diff
@@ -47,4 +50,20 @@ class ApplicationModelSpec extends Specification {
     listApp.cloudProviders == 'a,b'
     normalApp.cloudProviders == 'a,b,c'
   }
+
+  void 'ignore correct timestamped json properties from application object to json'() {
+
+    given:
+    def application = new Application()
+    application.setCreatedAt(new Long(1))
+    application.setLastModifiedBy("foo")
+
+    ObjectMapper mapper = new ObjectMapper().addMixIn(Timestamped.class, TimestampedMixins.class)
+    String appJSON = mapper.writeValueAsString(application)
+
+    expect:
+    String expectJSON = '{"name":null,"description":null,"email":null,"updateTs":null,"createTs":"1","lastModifiedBy":"foo","cloudProviders":null,"trafficGuards":[]}'
```

Comment:
I think you can get rid of this assignment and just make the assertion below with the string literal.